src/utils/urlPaths.ts

```diff
@@ -23,6 +23,7 @@ export enum UrlPathTemplate {
   Logout = '/logout',
   MyPage = '/my-page',
   MyPageMessages = '/my-page/messages',
+  MyPageMyFieldAndBackground = '/my-page/my-profile/my-field-and-background',
```

Comment:
Begynner å bli en heftig URL med mye `my` dette 😅 Kanskje bare bruke `(...)/background` elns i stedet?
Kunne nok endret `my-profile` til bare `profile` for å få det enklere også, men det er en annen sak.